purchase_discount/tests/test_purchase_discount.py

```diff
@@ -11,10 +12,14 @@ def setUp(self):
         super(TestPurchaseOrder, self).setUp()
         self.product_1 = self.env.ref('product.product_product_4')
         self.product_2 = self.env.ref('product.product_product_5b')
+        purchase_list = self.env['product.pricelist'].create({
+            'name': 'Default Purchase Pricelist',
+            'type': 'purchase',
+        })
         po_model = self.env['purchase.order.line']
         self.purchase_order = self.env['purchase.order'].create(
             {'partner_id': self.env.ref('base.res_partner_3').id,
-             'pricelist_id': self.env.ref('purchase.list0').id,
+             'pricelist_id': purchase_list.id,
```

Comment:
In 9.0 pricelist_id field has been droped
https://github.com/OCA/OCB/commit/e4dc50bf58d508d95b4ce9909af7e0ddfa2e46cf#diff-fe8587929089e5fdb6c75687011b1c7eL229